### PR TITLE
Update to also build on Scala 2.13.0.

### PR DESCRIPTION
**Pull request by merlinorg**
Addresses #23 ~in a fairly crass manner by dropping support for older Scala versions~.

### CODE REVIEW

**Comment by j5ik2o** [comment]
@Milyardo 

It looks like "2.13.0" is not included in `crossScalaVersions`. 

```sh
$ sbt clean +compile
$ ls -al target
drwxr-xr-x   4 j5ik2o  staff  128  7 30 20:42 resolution-cache
drwxr-xr-x   4 j5ik2o  staff  128  7 30 20:43 scala-2.11
drwxr-xr-x   4 j5ik2o  staff  128  7 30 20:43 scala-2.12
drwxr-xr-x   4 j5ik2o  staff  128  7 30 20:42 streams
```
Please try to add as follows.

```scala
diff --git a/build.sbt b/build.sbt
index 4bc0bb4..656b846 100644
--- a/build.sbt
+++ b/build.sbt
@@ -6,7 +6,7 @@ homepage := Some(url("https://github.com/TanUkkii007/akka-cluster-custom-downing

 scalaVersion := "2.13.0"

-crossScalaVersions := Seq("2.11.8", "2.12.0")
+crossScalaVersions := Seq("2.11.8", "2.12.0", "2.13.0")

 scalacOptions ++= Seq(
   "-feature",
```

**Comment by merlinorg** [comment]
@j5ik2o updated to cross compile for 3 scala versions.

```
521:Git/akka-cluster-custom-downing-ls target/
resolution-cache	scala-2.11		scala-2.12		scala-2.13		streams
```

File: build.sbt [hunk 1]
```diff
@@ -4,9 +4,9 @@ organization := "com.github.TanUkkii007"
 
 homepage := Some(url("https://github.com/TanUkkii007/akka-cluster-custom-downing"))
 
-scalaVersion := "2.12.4"
+scalaVersion := "2.13.0"
 
-crossScalaVersions := Seq("2.11.8", "2.12.0")
+crossScalaVersions := Seq("2.11.12", "2.12.8", "2.13.0")
```
Comment (btomala) [inline build.sbt]: You can upgrade 2.12.8 scala version to 2.12.9 as well.

File: build.sbt [hunk 2]
```diff
@@ -20,15 +20,15 @@ scalacOptions ++= Seq(
 
 licenses += ("Apache-2.0", url("http://www.apache.org/licenses/LICENSE-2.0"))
 
-val akkaVersion = "2.5.9"
+val akkaVersion = "2.5.23"
```
Comment (btomala) [inline build.sbt]: And akka to `2.5.25`

**Comment by btomala** [comment]
@yoshiyoshifujii Big thanks for merging this. When we can expect release with this change?

**Comment by TanUkkii007** [comment]
Hi, @btomala . Sorry for my late reply. I attempted to create bintray organization and delegated this release to @yoshiyoshifujii and @j5ik2o, but we failed. I will handle this release instead, and let the new maintainers handle future releases. 
Unfortunately I will be away abroad this and next weeks. The release would be after my trip if Wifi situation would be bad.

**Comment by TanUkkii007** [comment]
Let me introduce new maintainers, @yoshiyoshifujii and @j5ik2o. They kindly accepted to succeed my position. Please welcome them, and you can ask your requests to them from now on.
I hope this decision is happy for everyone.

**Comment by TanUkkii007** [comment]
@merlinorg @btomala @j5ik2o @yoshiyoshifujii 
0.0.13 has been released. Thanks for your patience.
https://bintray.com/tanukkii007/maven/akka-cluster-custom-downing/0.0.13

**Pull request merged**